src/bundle/msi_bundle.rs

```diff
@@ -429,9 +464,9 @@ fn create_feature_table(package: &mut Package, settings: &Settings) -> crate::Re
         msi::Value::from(settings.bundle_name()),
         msi::Value::Null,
         msi::Value::Int(1),
-        msi::Value::Int(3),
+        msi::Value::Int(1),
```

Comment:
With Feature Level 3 and `INSTALLLEVEL` 1, this feature will never be installed.